README.md

```diff
@@ -24,9 +24,9 @@ monitor progress, and troubleshoot issues when needed.
 
 ## Getting started
 
-Please visit the Airflow Platform documentation (latest **stable** release) for help with [installing Airflow](https://airflow.incubator.apache.org/installation.html), getting a [quick start](https://airflow.incubator.apache.org/start.html), or a more complete [tutorial](https://airflow.incubator.apache.org/tutorial.html).
+Please visit the Airflow Platform documentation (latest **stable** release) for help with [installing Airflow](https://airflow.apache.org/installation.html), getting a [quick start](https://airflow.apache.org/start.html), or a more complete [tutorial](https://airflow.apache.org/tutorial.html).
```

Comment:
@tedmiston I've granted you perms on wiki. Do you mind also updating https://cwiki.apache.org/confluence/display/AIRFLOW/Building+and+deploying+the+docs?

There's a line there: `This should point to latest stable, while readthedocs.org keeps track of versioned documentation.`.

We should mention both here and in the wiki:
1. Where latest master docs reside
2. Where latest stable docs reside
3. Where release-version docs reside

I suspect that `1` above is not realistic since master is always a moving target, so we should at make the wording consistent in how we describe the other 2 cases.

Reply:
Also, this doc here mentions that latest master is on readthedocs? Are the docs being built by some github webhook when we merge PRs to master? If not, how do we ensure the readthedocs reflects master?

Reply:
@r39132 Hi Sid, we had discussed this at the start of this year about using the webhooks but as we didn't have permissions to create hooks on Github, the current process is manual i.e. I try and trigger the build manually regularly.

Reply:
We can open an ticket with Apache INFRA to enable a webhook for this.

Reply:
Cool. I will raise this today.

Reply:
I have raised the issue with Apache Infra over here: https://issues.apache.org/jira/browse/INFRA-16895

Reply:
It appears from the issue linked above that Apache Infra added this webhook on 8/11/13.

@kaxil Can you see whether it's setup and working from the RTD admin side?

Reply:
Yes, it is setup and working.